### PR TITLE
[TASK] Make compatible with Composer 2.0

### PR DESCRIPTION
**Pull request by gilbertsoft**
Related https://github.com/TYPO3-Console/TYPO3-Console/pull/944

### CODE REVIEW

**Comment by gilbertsoft** [comment]
Should get squash merged!

File: composer.json [hunk 1]
```diff
@@ -22,10 +22,10 @@
     "require": {
         "php": ">=5.5.0",
         "typo3/cms-composer-installers": "^1.3.1",
```
Comment (gilbertsoft) [inline composer.json:24]: Not sure if we also need newer versions here

**Comment by helhum** [comment]
Merging now without testing. Please report back, whether it works @YetiCGN

**Comment by YetiCGN** [comment]
> Merging now without testing. Please report back, whether it works @YetiCGN

Yup! Looks good with 7839aee. 🥳

**Pull request merged**